en/stacks/audio/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Audio management
+---
+
+# Audio management
+
+There are currently two different snaps that help manage audio under Ubuntu Core Stacks:
+
+1. alsa-utils
+2. pulseaudio
```

Comment:
Is there a need for a numbered list here (eg. steps to follow or order of preference)? If not, please use a simple list (same goes for other pages)